Lab2/Lab2.ipynb

```diff
@@ -325,7 +316,9 @@
    "id": "d11db467-8dd3-4cc1-83f9-3a7800335030",
    "metadata": {},
    "source": [
-    "We see that as we sum and average the background with itself over and over, the standard deviation becomes smaller and the distribution more narrow (even though, visibly this is not immediately apparent - therefore, look at the scaled y-axis)."
+    "We see that as we sum and average the background with itself over and over, the standard deviation becomes smaller and the distribution more narrow (even though, visibly this is not immediately apparent - therefore, look at the scaled y-axis).\n",
```

Comment:
The graphs here are just a little difficult to interpret on first glance in my opinion, maybe adjusting the x-limits could help? Just because it looks a little "overflowing" with the plot taking up so much of the graph, if that makes sense.